iff --git a/patches/protonprep-valve-staging.sh b/patches/protonprep-valve-staging.sh
index 635c9395..fc86209c 100755
--- a/patches/protonprep-valve-staging.sh
+++ b/patches/protonprep-valve-staging.sh
@@ -6,34 +6,7 @@
 # I was pinged about it from DXVK dev discord.
 # https://github.com/doitsujin/dxvk/pull/3690#discussion_r1405306492
 
-    pushd dxvk
-    git reset --hard HEAD
-    git clean -xdf
-    #echo "DXVK: -Nvidia Reflex- Add NV low latency support"
-    #pushd include/vulkan; git pull; git checkout bbe0f575ebd6098369f0ac6c6a43532732ed0ba6; popd
-    #patch -Np1 < ../patches/proton/80-nv_low_latency_dxvk.patch
-    popd
-
-    pushd vkd3d-proton
-    git reset --hard HEAD
-    git clean -xdf
-    
-    #echo "VKD3D-PROTON: -Nvidia Reflex- Add NV low latency support"
-    #pushd khronos/Vulkan-Headers; git pull; git checkout bbe0f575ebd6098369f0ac6c6a43532732ed0ba6; popd
-    #patch -Np1 < ../patches/proton/81-nv_low_latency_vkd3d_proton.patch
-    popd
-
-    pushd dxvk-nvapi
-    git reset --hard HEAD
-    git clean -xdf
-    #echo "DXVK-NVAPI: -Nvidia Reflex- Add support for Reflex"
-    #patch -Np1 < ../patches/proton/82-nv_low_latency_dxvk_nvapi.patch
-    popd
-
     pushd gstreamer
-    git reset --hard HEAD
-    git clean -xdf
-    
     echo "GSTREAMER: fix for unclosable invisible wayland opengl windows in taskbar"
     patch -Np1 < ../patches/gstreamer/5509.patch
     patch -Np1 < ../patches/gstreamer/5511.patch
@@ -44,15 +17,14 @@
 ### (2) WINE PATCHING ###
 
     pushd wine
-    git reset --hard HEAD
-    git clean -xdf
 
 ### (2-1) PROBLEMATIC COMMIT REVERT SECTION ###
 
 # Bring back configure files. Staging uses them to regenerate fresh ones
 # https://github.com/ValveSoftware/wine/commit/e813ca5771658b00875924ab88d525322e50d39f
 
-    git revert --no-commit e813ca5771658b00875924ab88d525322e50d39f
+    # git revert --no-commit e813ca5771658b00875924ab88d525322e50d39f
+    # umubuilder: done in a separate patch!
 
 ### END PROBLEMATIC COMMIT REVERT SECTION ###

